projects/gnome.org/PyGObject/package.yml

```diff
@@ -0,0 +1,40 @@
+distributable:
+  url: https://download.gnome.org/sources/pygobject/{{version.marketing}}/pygobject-{{version}}.tar.xz
+  strip-components: 1
+versions:
+  url: https://download.gnome.org/sources/pygobject/cache.json
+  match: /pygobject-\d+\.\d+\.\d+\.tar\.xz/
+  strip:
+    - /^pygobject-/
+    - /\.tar\.xz/
+dependencies:
+  gnome.org/gobject-introspection: '*'
+  cairographics.org/pycairo: '*'
+runtime:
+  env:
+    PYTHONPATH: "{{prefix}}/lib/python3.11/site-packages:$PYTHONPATH"
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    mesonbuild.com: '*'
+    ninja-build.org: '*'
+    freedesktop.org/pkg-config: '*'
+    python.org: ~3.11
```

Comment:
although if we leave it hard-locked to 3.11 (which probably isn't right), it won't matter.